bin/minify.js

```diff
@@ -12,8 +14,8 @@ const log = function(...args) {
 };
 
 const Argv = process.argv;
-const files = Argv.slice(2);
-const [In] = files;
+const args = yargsParser(Argv.slice(2));
```

Comment:
Could you please put `yargs-parser` to `lib/cli.js` similar to [yargs-parse example](https://github.com/coderaiser/putout/blob/master/packages/putout/lib/cli/index.js#L82-L139)